packages/pipeline/src/entities/ohlcv_external.ts

```diff
@@ -0,0 +1,20 @@
+import { Column, Entity, PrimaryColumn } from 'typeorm';
+
+@Entity({ name: 'ohlcv_external', schema: 'raw' })
+export class OHLCVExternal {
+    @PrimaryColumn() public exchange!: string;
+    @PrimaryColumn() public fromSymbol!: string;
+    @PrimaryColumn() public toSymbol!: string;
+    @PrimaryColumn() public startTime!: number;
+    @PrimaryColumn() public endTime!: number;
+
+    @Column() public open!: number;
```

Comment:
Should we use the `BigNumber` type here just to be safe? `number` can sometimes lead to precision issues.

Reply:
number for prices is fine since these are not precision sensitive generally, but bignumber for times is probably a good standard practice since those have like 15 digits. i think we should do this in a separate PR since it applies to all of our timestamp entities?

Reply:
<as per discussion, timestamp ms has plenty of runway b/c number has 15 decimals of precision vs 12 decimals of ms since 1970, and pricing data via cryptocompare and similar api's are not precision-sensitive nor reported as long strings of precise values>